packages/cli/subtasks/pick-function.js

```diff
@@ -16,9 +16,7 @@ subtask(SUBTASK_PICK_FUNCTION, 'Pick a function from the given contract').setAct
       const selector = selectors.find((selector) => selector.name === functionAbi.name).selector;
 
       return {
-        title: `${chalk.gray(`${hre.cli.contractName}.`)}${fullSignature}${chalk.gray(
-          ` ${selector}`
-        )}`,
+        title: `${fullSignature}${chalk.gray(` ${selector}`)}`,
```

Comment:
Looks like prompter lib autocomplete is not that good. E.g. when typing "set" it will not pick "SomeModule.setUintValue" but will pick "setUintValue" (when removing contract name).